app/Models/Training/WaitingListAccount.php

```diff
@@ -165,4 +165,13 @@ public function addNote($contents)
 
         return $this->notes;
     }
+
+    public function editNote($contents)
```

Comment:
Should this perhaps be renamed `editNotes` in line with the field name? Also, is there a specific reason why we would want this as a function rather than just a set mutator? At the moment, might just be for me, but when I see `addNote` and `editNote` it makes me think that it is a cumulative add (when it might not be?)

Reply:
New requirements dictate that we aren't going to distinguish between editing and creating a note, so expect a change shortly.